packages/usa-file-input/src/index.js

```diff
@@ -121,20 +133,6 @@ const makeSafeForID = (name) => name.replace(/[^a-z0-9]/g, replaceName);
 const createUniqueID = (name) =>
   `${name}-${Math.floor(Date.now().toString() / 1000)}`;
 
-/**
- * Determines if the singular or plural item label should be used
- * Determination is based on the presence of the `multiple` attribute
- *
- * @param {HTMLInputElement} fileInputEl - The input element.
- * @returns {HTMLDivElement} The singular or plural version of "item"
- */
-const getItemsLabel = (fileInputEl) => {
-  const acceptsMultiple = fileInputEl.hasAttribute("multiple");
-  const itemsLabel = acceptsMultiple ? "files" : "file";
-
-  return itemsLabel;
-};
-
```

Comment:
> **Note**
This was removed because we now are requiring full phrase strings to accommodate unpredictable sentence structure across languages.